src/grid/Row/index.style.js

```diff
@@ -0,0 +1,217 @@
+import styled from "styled-components";
+
+const RowStyles = styled.div`
+  .grid {
+    display: flex;
+  }
+
+  .row {
+    box-sizing: border-box;
+    display: flex;
+    flex: 0 1 auto;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .row.reverse {
+    flex-direction: row-reverse;
```

Comment:
can this not be 

`css
.row {
  .reverse {
    flex-direction: row-reverse
  }
}
`

Reply:
i tried it didnt work out.